beangulp/testing.py

```diff
@@ -46,7 +46,7 @@ def write_expected_file(filepath: str, *data, force: bool = False):
     """Writes out the expected file."""
     mode = 'w' if force else 'x'
     with open(filepath, mode) as expfile:
-        write_expected(expfile, *data)
+        write_expected(expfile, *data)  # type: ignore
```

Comment:
Why?

Reply:
expfile is `IO[Any]` and write_expected takes `TextIO` - not really sure why mypy doesn't infer that it's in text mode